### PR TITLE
improve yaml context for storage/storage-classes.md

### PR DESCRIPTION
**Pull request by my-git9**
content/en/docs/concepts/storage/storage-classes.md

### CODE REVIEW

File: content/en/docs/concepts/storage/storage-classes.md [hunk 1]
```diff
@@ -298,7 +299,7 @@ allowedTopologies:
     values:
     - us-east-2c
```
Comment (tengqm) [inline content/en/docs/concepts/storage/storage-classes.md:301]:
````suggestion
```
````

File: content/en/docs/concepts/storage/storage-classes.md [hunk 2]
```diff
@@ -278,6 +278,7 @@ The Kubernetes project suggests that you use the [AWS EBS](https://github.com/ku
 out-of-tree storage driver instead.
 
 Here is an example StorageClass for the AWS EBS CSI driver:
+
 ```yaml
```
Comment (sftim) [inline content/en/docs/concepts/storage/storage-classes.md:282]: Could we use a `code_sample` shortcode here? See https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#source-code-files

Comment (my-git9) [reply content/en/docs/concepts/storage/storage-classes.md:282]: Good idea

File: content/en/examples/storage/storageclass/storageclass-azure-file.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: azurefile
+provisioner: kubernetes.io/azure-file
+parameters:
+  skuName: Standard_LRS
+  location: eastus
+  storageAccount: azure_storage_account_name
```
Comment (sftim) [inline content/en/examples/storage/storageclass/storageclass-azure-file.yaml:9]: You could even
```suggestion
  storageAccount: azure_storage_account_name # example value
```

File: content/en/examples/storage/storageclass/storageclass-ceph-rbd.yaml [hunk 4]
```diff
@@ -0,0 +1,17 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: fast
+provisioner: kubernetes.io/rbd
+parameters:
+  monitors: 10.16.153.105:6789
```
Comment (sftim) [inline content/en/examples/storage/storageclass/storageclass-ceph-rbd.yaml:7]: Optionally, switch it to a documentation IP address
```suggestion
  monitors: 198.19.254.105:6789
```

File: content/en/examples/storage/storageclass/storageclass-ceph-rbd.yaml [hunk 5]
```diff
@@ -0,0 +1,17 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: fast
+provisioner: kubernetes.io/rbd
```
Comment (sftim) [inline content/en/examples/storage/storageclass/storageclass-ceph-rbd.yaml:5]:
```suggestion
provisioner: kubernetes.io/rbd # This provisioner is deprecated
```

File: content/en/examples/storage/storageclass/storageclass-local.yaml [hunk 6]
```diff
@@ -0,0 +1,6 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: local-storage
+provisioner: kubernetes.io/no-provisioner
```
Comment (sftim) [inline content/en/examples/storage/storageclass/storageclass-local.yaml:5]: Is this a real provisioner identifier?
If so, I'd add a brief comment.

File: content/en/examples/storage/storageclass/storageclass-portworx-volume.yaml [hunk 7]
```diff
@@ -0,0 +1,9 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: portworx-io-priority-high
+provisioner: kubernetes.io/portworx-volume
```
Comment (sftim) [inline content/en/examples/storage/storageclass/storageclass-portworx-volume.yaml:5]: How about:
```suggestion
provisioner: kubernetes.io/portworx-volume # # This provisioner is deprecated
```

File: content/en/examples/storage/storageclass/storageclass-topology.yaml [hunk 8]
```diff
@@ -0,0 +1,14 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: standard
+provisioner: kubernetes.io/example
```
Comment (sftim) [inline content/en/examples/storage/storageclass/storageclass-topology.yaml:5]: Maybe also:
```suggestion
provisioner: example.com/example
```

**Review by sftim — COMMENTED** [review]
Optionally, tweak the sample manifests further.

**Review by sftim — COMMENTED** [review]
Thanks

/lgtm

**Review by dipesh-rawat — COMMENTED** [review]
/approve

**Pull request merged**